jekyll/_posts/2021-01-19-community-survey-results-2020.md

```diff
@@ -0,0 +1,221 @@
+We are happy to see that 2/3 of our users plan to do it in the future, which will help us to have more manpower to fix the most important bugs more promptly and work on some exciting new features.
+
+
+
+
+
+## Nim tools
+
+We've asked you how satisfied you are with the Nim tooling:
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/29.png" alt="What is your experience with the following tools? [nimble]" style="width:100%"/>
+<img src="{{site.baseurl}}/assets/news/images/survey2020/30.png" alt="What is your experience with the following tools? [choosenim]" style="width:100%"/>
+
+Most of our users use Nimble package manager and they are very satisfied by it.
+A smaller amount of users picked Choosenim as their way of installing and updating Nim versions, but those who did are quite satisfied with what it has to offer.
+
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/34.png" alt="What is your experience with the following tools? [nim doc]" style="width:100%"/>
+<img src="{{site.baseurl}}/assets/news/images/survey2020/33.png" alt="What is your experience with the following tools? [nimpretty]" style="width:100%"/>
+<img src="{{site.baseurl}}/assets/news/images/survey2020/32.png" alt="What is your experience with the following tools? [nimsuggest]" style="width:100%"/>
+
+Nim doc is generally liked by those who use it, as is Nimpretty.
+Nimsuggest is liked a bit less than other tools (high CPU and memory consumption), but you'll be glad to hear that our plans for 2021 should also make Nimsuggest more robust and less CPU and memory hungry.
+
+
+
+
+
+## Nim in 2021 and beyond
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/24.png" alt="What should be our priorities in improving Nim, that would bring the largest quality-of-life improvements? [fixing compiler bugs]" style="width:100%"/>
+
+According to the votes, fixing compiler bugs should be our first priority; 55% of our users find it very important, with only 18% of votes for ok or low priority.
+We will continue to work on it, and we appreciate any community effort on this: we already have several individuals continuously helping us with fixing compiler bugs (thank you!), and we'd benefit from more.
+
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/22.png" alt="What should be our priorities in improving Nim, that would bring the largest quality-of-life improvements? [improve tooling]" style="width:100%"/>
+
+Improving the tooling is very high on a priority list for our users (higher than in 2019), and especially for those users who use Nim professionally.
+We're happy to announce that, thanks to our partnership with [Status](https://status.im), improving the developer tooling is [one of the goals for 2021](https://github.com/nim-lang/RFCs/issues/300).
```

Comment:
This should be detailed. I hope people don't assume that I will be tackling developer tooling.

Reply:
From what I've heard, Status will employ somebody (not you) to work on VS Code extension as their full-time job.